packages/url/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@rollup/plugin-url",
+  "version": "3.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Import files as data-URIs or to ES6 modules.",
+  "license": "MIT",
+  "repository": "rollup/plugins",
+  "author": "Arpad Borsos <arpad.borsos@googlemail.com>",
+  "homepage": "https://github.com/rollup/plugins/packages/url/#readme",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "dist/index.js",
+  "engines": {
+    "node": ">=10.0.0"
```

Comment:
Rollup is still supporting `>= 8.0.0`

Reply:
@pnevares I will take care of this. can you give me permission to push to repo?

Reply:
@Ashot-KR I've invited you as a collaborator on my fork 👍

Reply:
Fixed. Plugin will work with node >=8